lib/Sema/ResilienceDiagnostics.cpp

```diff
@@ -176,6 +177,11 @@ bool TypeChecker::diagnoseInlinableDeclRefAccess(SourceLoc loc,
     diagName = accessor->getStorage()->getFullName();
   }
 
+  // Swift 5.0 did not check the underlying types of local typealiases.
+  // FIXME: Conditionalize this once we have a new language mode.
```

Comment:
Still worried about these. This is the second one I'm adding. Should we be using `isSwiftVersionAtLeast(9999)` in the mean time?